src/Microsoft.AspNetCore.Mvc.Core/DisableRequestSizeLimitAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// Sets the <see cref="IHttpMaxRequestBodySizeFeature.MaxRequestBodySize"/> to <c>null</c>.
```

Comment:
I hadn't seen @Tratcher's comment on the feature when I made my earlier common on the inconsistent `<see/>` versus descriptive text. Best to consistently describe what's going on here in the public classes. But, leave what you've got in the `.Internal` classes alone.

> The feature interfaces aren't really meant for direct user consumption. putting it in the docs here would be more confusing than the current description.